src/main/java/com/mabl/integration/jenkins/MablRestApiClientImpl.java

```diff
@@ -223,6 +243,23 @@ public String getAppBaseUrl() {
         return appBaseUrl;
     }
 
+    @Override
+    public void checkConnection() throws IOException {
+        final String url = restApiBaseUrl + HEALTH_LIVE_ENDPOINT;
+        final HttpResponse response = httpClient.execute(buildGetRequest(url));
+        final int statusCode = response.getStatusLine().getStatusCode();
+        switch (statusCode) {
+            case 200:
+                break;
+            case 401:
+                throw new IOException("Invalid API key provided or the proxy connection requires authentication");
+            case 403:
+                throw new IOException("Invalid API key type. You must use a CI/CD Integration key type");
```

Comment:
```suggestion
                throw new IOException("Invalid API key type. You must use a 'CI/CD Integration' key type");
```

Maybe we add quotes to help them realize this is the explicit key type name.